### PR TITLE
Spike and Slab tutorial with PyMC

### PR DESCRIPTION
**Pull request by GianmarcoCallegher**

### CODE REVIEW

**Comment by GianmarcoCallegher** [comment]
@jobrachem it can be reviewed. I don't know if you really want to go through the code. I don't think it is necessary

File: spike-and-slab.ipynb [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import liesel.goose as gs\n",
+    "import numpy as np\n",
+    "import pymc as pm\n",
```
Comment (wiep) [inline spike-and-slab.ipynb:11]: this file is probably in the repo by accident.

**Comment by jobrachem** [comment]
@GianmarcoCallegher Thanks, I'll do the review soon :)

**Review by jobrachem — APPROVED** [review]

**Comment by jobrachem** [comment]
@wiep you are registered as a reviewer here, too. Are you ok with merging this PR?

**Pull request merged**